From d6b50428f1449e9beb8c8eff4ee5f639a07b7485 Mon Sep 17 00:00:00 2001
From: Heinrich Schuchardt <xypron.glpk@gmx.de>
Date: Wed, 6 Sep 2017 17:48:26 +0200
Subject: [PATCH 1/1] vsprintf: vsprintf does not have parameter size

The inline documentation of vsprintf mentions a parameter size
which does not exist in the function declaration.

int vsprintf(char *buf, const char *fmt, va_list args);

Signed-off-by: Heinrich Schuchardt <xypron.glpk@gmx.de>
---
 include/vsprintf.h | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/include/vsprintf.h b/include/vsprintf.h
index e38076d22d..490c96ca6d 100644
--- a/include/vsprintf.h
+++ b/include/vsprintf.h
@@ -112,12 +112,10 @@ int sprintf(char *buf, const char *fmt, ...)
  * Format a string and place it in a buffer (va_list version)
  *
  * @param buf	The buffer to place the result into
- * @param size	The size of the buffer, including the trailing null space
  * @param fmt	The format string to use
  * @param args	Arguments for the format string
  * @return the number of characters which have been written into
- * the @buf not including the trailing '\0'. If @size is == 0 the function
- * returns 0.
+ * the @buf not including the trailing '\0'.
  *
  * If you're not already dealing with a va_list consider using scnprintf().
  *
-- 
2.11.0

